python/cuml/tests/test_module_config.py

```diff
@@ -1,130 +0,0 @@
-#
```

Comment:
The tests in this file passed fine in the new system (if you rollback a few commits you can run them).

I migrated the intentions (but not necessarily the implementation) to a more complete set of tests in `test_reflection.py`.